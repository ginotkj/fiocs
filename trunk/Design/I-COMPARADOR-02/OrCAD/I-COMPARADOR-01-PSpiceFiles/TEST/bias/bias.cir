** Profile: "TEST-bias"  [ D:\Documents\TESIS\TRUNK\Design\I-COMPARADOR-01\OrCAD\I-COMPARADOR-01-PSpiceFiles\TEST\bias.sim ] 

** Creating circuit file "bias.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../i-comparador-01-pspicefiles/i-comparador-01.lib" 
* From [PSPICE NETLIST] section of C:\OrCAD\OrCAD_10.5\tools\PSpice\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.PROBE V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\TEST.net" 


.END
