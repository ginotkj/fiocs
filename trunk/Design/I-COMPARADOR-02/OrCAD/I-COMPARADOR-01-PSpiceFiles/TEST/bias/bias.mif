 lib=D:\Documents\TESIS\fiocs\Design\I-COMPARADOR-02\OrCAD\I-COMPARADOR-01-PSpiceFiles\i-comparador-01.lib, offset=2473, size=2424
 lib=D:\Documents\TESIS\fiocs\Design\I-COMPARADOR-02\OrCAD\I-COMPARADOR-01-PSpiceFiles\i-comparador-01.lib, offset=42, size=2430
