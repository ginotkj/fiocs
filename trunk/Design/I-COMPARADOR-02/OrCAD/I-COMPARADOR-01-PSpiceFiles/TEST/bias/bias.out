
**** 01/19/10 20:31:18 ******* PSpice 10.5.0 (Jan 2005) ******* ID# 0 ********

 ** Profile: "TEST-bias"  [ D:\Documents\TESIS\TRUNK\Design\I-COMPARADOR-01\OrCAD\I-COMPARADOR-01-PSpiceFiles\TEST\bias.sim ] 


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "bias.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../i-comparador-01-pspicefiles/i-comparador-01.lib" 
* From [PSPICE NETLIST] section of C:\OrCAD\OrCAD_10.5\tools\PSpice\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.PROBE V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\TEST.net" 



**** INCLUDING TEST.net ****
* source I-COMPARADOR-01
M_M2         VDIFERENCIAL IN+ VEQUILIBRIO 0 CMOSN  
+ L=1u  
+ W=2u         
V_VREF         IN+ N398131 150m
M_M3         N39461 N39461 N39211 N39211 CMOSP  
+ L=3u  
+ W=6u         
M_M4         VDIFERENCIAL N39461 N39211 N39211 CMOSP  
+ L=3u  
+ W=6u         
M_M5         VEQUILIBRIO VBIAS 0 0 CMOSN  
+ L=1u  
+ W=2u         
V_VQ+         N398131 0 .2
V_VIN         IN- N43113  
+SIN 0 200m 1Meg 0 0 0
M_M6         OUT VDIFERENCIAL N39211 N39211 CMOSP  
+ L=2u  
+ W=8u         
V_VBIAS         VBIAS 0 .5
M_M7         OUT VBIAS 0 0 CMOSN  
+ L=1u  
+ W=2u         
V_VQ-         N43113 0 .2
M_M1         N39461 IN- VEQUILIBRIO 0 CMOSN  
+ L=1u  
+ W=2u         
V_VDD         N39211 0 5

**** RESUMING bias.cir ****
.END

WARNING -- Mosfet M_M2, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_M2, model CMOSN: Ps = 0 is less than W
WARNING -- Mosfet M_M3, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_M3, model CMOSP: Ps = 0 is less than W
WARNING -- Mosfet M_M6, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_M6, model CMOSP: Ps = 0 is less than W
**** 01/19/10 20:31:18 ******* PSpice 10.5.0 (Jan 2005) ******* ID# 0 ********

 ** Profile: "TEST-bias"  [ D:\Documents\TESIS\TRUNK\Design\I-COMPARADOR-01\OrCAD\I-COMPARADOR-01-PSpiceFiles\TEST\bias.sim ] 


 ****     MOSFET MODEL PARAMETERS


******************************************************************************




               CMOSN           CMOSP           
               NMOS            PMOS            
  T_Measured   27              27            
   T_Current   27              27            
       LEVEL    7               7            
           L  100.000000E-06  100.000000E-06 
           W  100.000000E-06  100.000000E-06 
         VTO     .299302        -.401632     
          KP  470.883200E-06  470.883200E-06 
       GAMMA    0               0            
      LAMBDA    0               0            
         RSH    6               6.4          
          IS    1.000000E-15    1.000000E-15 
          JS  100.000000E-06  100.000000E-06 
          PB     .980128         .821464     
        PBSW     .8              .8008       
          CJ  815.376400E-06    1.165977E-03 
        CJSW  166.520200E-12  122.005600E-12 
          MJ     .606194         .425655     
        MJSW     .169221         .1001       
        CGSO  500.000000E-12  500.000000E-12 
        CGDO  500.000000E-12  500.000000E-12 
        CGBO    1.000000E-12    1.000000E-12 
         TOX    4.400000E-09    4.400000E-09 
          XJ  100.000000E-09  100.000000E-09 
       UCRIT   10.000000E+03   10.000000E+03 
       DELTA     .01             .01         
      DIOMOD    2               2            
          K1     .542117         .598774     
          K2    -.035133     -630.087300E-09 
        LETA    0               0            
        WETA    0               0            
          U0  268.8192        120.6045       
       XPART     .5              .5          
        VTH0     .299302        -.401632     
          K3    1.000000E-03     .081248     
          W0  100.000000E-09    1.000000E-06 
         NLX  228.107500E-09   42.193340E-09 
        DVT0     .882304         .948505     
        DVT1     .324768         .682461     
          UA   -1.551592E-09    1.640708E-09 
          UB    2.710709E-18    1.272651E-21 
          UC   46.128640E-12 -100.000000E-12 
        VSAT  116.006100E+03  182.237700E+03 
        RDSW  150             726.9944       
        VOFF    -.091862        -.15         
     NFACTOR    1.807002         .921842     
        PCLM    2.027608         .151904     
      PDIBL1     .784214         .056431     
      PDIBL2    -.042689         .012341     
       DROUT    1                .470272     
      PSCBE1    2.314588E+09    4.022317E+09 
      PSCBE2  500.000000E-12    7.751373E-09 
          A0    1.525088         .904176     
          A1    6.962179E-03    8.069322E-03 
          A2     .524367         .560796     
       NPEAK  235.490000E+15  415.890000E+15 
         LDD    0               0            
        LITL   36.331810E-09   36.331810E-09 
         UC1  -56.000000E-12  -56.000000E-12 
        PVAG                     .124556     
        KETA    -.015366         .020751     
        ETA0    5.535422E-03    1.016564E-03 
        ETAB -671.692500E-06   -1.910199E-03 
         K3B    5.015682       19.7189       
        DVT2    -.056267        -.3          
        DSUB    3.692569E-03     .014544     
         AGS     .400988         .131884     
       DVT1W    0               0            
       DVT2W    0               0            
        PRWG     .340675        -.049475     
        PRWB    -.183865        -.420982     
     PDIBLCB    -.1            -1.000000E-03 
         DWG    5.852475E-09  -30.588980E-09 
         DWB   11.572970E-09  -11.142310E-09 
          B0  -44.157810E-09    1.909663E-06 
          B1                    5.000000E-06 
        LINT   20.701890E-09   26.472300E-09 
        WINT   12.149120E-09                 
         DLC   20.701890E-09   26.472300E-09 
         DWC   12.149120E-09    0            
          CF    0               0            
        NOIA  100.000000E+18    9.900001E+18 
        NOIB   50.000000E+03    2.400000E+03 
        NOIC   -1.400000E-12    1.400000E-12 
       LKETA   -1.853360E-03   -8.835292E-03 
       WKETA    3.404068E-03     .030613     
       PVSAT    1.045662E+03   50            
       PKETA    -.0116         -5.087778E-03 
       PVTH0   -3.570294E-03  633.898700E-06 
         PUA  -25.147800E-12  364.727200E-12 
         PUB   21.902820E-24                 
         PU0   -2.447231        7.159787     
       PRDSW   -1.670209       -5            
       PETA0 -100.000000E-06  100.000000E-06 
         PK2    3.193075E-03   31.405140E-06 
         VTM     .025864         .025864     
       PBSWG     .8              .8008       
       MJSWG     .169221         .1001       
       CJSWG  330.000000E-12  422.000000E-12 


WARNING -- Mosfet M_M2, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_M2, model CMOSN: Ps = 0 is less than W
WARNING -- Mosfet M_M3, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_M3, model CMOSP: Ps = 0 is less than W
WARNING -- Mosfet M_M6, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_M6, model CMOSP: Ps = 0 is less than W
**** 01/19/10 20:31:18 ******* PSpice 10.5.0 (Jan 2005) ******* ID# 0 ********

 ** Profile: "TEST-bias"  [ D:\Documents\TESIS\TRUNK\Design\I-COMPARADOR-01\OrCAD\I-COMPARADOR-01-PSpiceFiles\TEST\bias.sim ] 


 ****     SMALL SIGNAL BIAS SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(  IN+)     .3500  (  IN-)     .2000  (  OUT)    4.9798  (VBIAS)     .5000      

(N39211)    5.0000 (N39461)    4.6727 (N43113)     .2000 (N398131)     .2000    

(VEQUILIBRIO)     .0015               (VDIFERENCIAL)     .0076              




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT

    V_VREF       0.000E+00
    V_VQ+        0.000E+00
    V_VIN        0.000E+00
    V_VBIAS      0.000E+00
    V_VQ-        0.000E+00
    V_VDD       -1.184E-05

    TOTAL POWER DISSIPATION   5.92E-05  WATTS



          JOB CONCLUDED

**** 01/19/10 20:31:18 ******* PSpice 10.5.0 (Jan 2005) ******* ID# 0 ********

 ** Profile: "TEST-bias"  [ D:\Documents\TESIS\TRUNK\Design\I-COMPARADOR-01\OrCAD\I-COMPARADOR-01-PSpiceFiles\TEST\bias.sim ] 


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =         .02
